pkg/sources/git/git.go

```diff
@@ -216,9 +217,11 @@ func CloneRepo(userInfo *url.Userinfo, gitUrl string) (clonePath string, repo *g
 		err = errors.WrapPrefix(err, "could not parse url", 0)
 		return
 	}
-
 	cloneURL.User = userInfo
-	cloneCmd := exec.Command("git", "clone", cloneURL.String(), clonePath)
+
+	gitArgs := []string{"clone", cloneURL.String(), clonePath}
```

Comment:
Found the issue - I accidentally constructed `gitArgs` before setting `cloneURL.User` here (fixed in latest commit). Strange that it didn't happen locally, but I'm glad the CI caught it :)